src/components/Main/MainView/MessageInput/MessageInputTextArea.vue

```diff
@@ -100,10 +101,16 @@ export default defineComponent({
 
     const { insertLineBreak } = useLineBreak(props, textareaRef, context)
 
-    const { onBeforeInput, onKeyDown, onKeyUp } = useSendKeyWatcher(
+    const { onBeforeInput, onKeyDownA, onKeyUp } = useSendKeyWatcher(
```

Comment:
```suggestion
    const { onBeforeInput, onKeyDown: onKeyDownSendKeyWatcher, onKeyUp } = useSendKeyWatcher(
```
こんな感じのがわかりやすいからこっちで :pray: